server/server.js

```diff
@@ -89,11 +89,21 @@ app.post(
   }
 );
 
+app.delete(
+  "/logout",
+  sessionController.terminateSession,
+  (req, res) => {
+    console.log("completing delete request to '/logout");
+    res.status(200).json(req.session);
```

Comment:
Does the front end need `req.session` for anything? If not, then

```suggestion
    res.sendStatus(200);
```

Reply:
Was used for testing purposes.  Suggested change implemented.